internal/pkg/mapper/orchestrator/remote_save_test.go

```diff
@@ -135,8 +137,8 @@ func TestMapBeforeRemoteSave(t *testing.T) {
 	}
 
 	// Save
-	assert.NoError(t, mapper.MapBeforeRemoteSave(recipe))
-	assert.Empty(t, logs.AllMessages())
+	assert.NoError(t, state.Mapper().MapBeforeRemoteSave(recipe))
+	assert.Empty(t, logger.WarnAndErrorMessages())
```

Comment:
Namiesto `AllMessages` -> `WarnAndErrorMessages`. 
Ako sa zmenil sposob vytvarania `dependencies`, tak tam pribudlo par `DEBUG` hlasok.
Je to na viacerych miestach v tomto commite.

No nas v teste nezaujimaju `debug` hlasky, ... ale to, ze nie je zalogovany ziadny warn/error.